src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,219 @@
+           left-section-icon-bg-color        colors/neutral-30
+           right-section-icons               [{:bg       colors/primary-40
+                                               :no-color true
+                                               :icon     :main-icons/placeholder20}]}}]
+  (let [{:keys [height width]}      (dimensions/window)
+        put-middle-section-on-left? (or (true? align-mid)
+                                        (> (count right-section-icons) 1))
+        mid-section-props {:mid-section-type                 mid-section-type
+                           :horizontal-description?          horizontal-description?
+                           :mid-section-main-text            mid-section-main-text
+                           :one-icon-align-left?             one-icon-align-left?
+                           :mid-section-right-icon           mid-section-right-icon
+                           :mid-section-icon                 mid-section-icon
+                           :mid-section-main-text-icon-color mid-section-main-text-icon-color
+                           :mid-section-left-icon            mid-section-left-icon}]
+    [rn/view {:style (cond->
+                      {:display            :flex
+                       :flex-direction     :row
+                       :width              width
+                       :height             (* 0.075 height)
+                       :align-items        :center
+                       :padding-horizontal 8
+                       :justify-content    :space-between}
+                       (string/blank? page-nav-background-uri) (assoc :background-color page-nav-color)
+                       (string/blank? page-nav-color) (assoc :background page-nav-background-uri))}
+     [rn/view {:style {:flex           1
+                       :flex-direction :row
+                       :align-items    :center}}
+      [rn/view {:style (merge
+                        icon-styles
+                        {:background-color left-section-icon-bg-color
+                         :width            32
+                         :height           32}
+                        (when put-middle-section-on-left? {:margin-right 5}))}
+       [icons/icon left-section-icon (icon-props left-section-icon-color :big)]]
+      (when put-middle-section-on-left?
+        [mid-section (merge {:left-align?                       true
+                             :mid-section-description           mid-section-description
+                             :mid-section-description-color     mid-section-description-color
+                             :mid-section-description-icon      mid-section-description-icon
+                             :mid-section-description-user-icon mid-section-description-user-icon}
+                            mid-section-props)])]
+     (when-not put-middle-section-on-left?
+       [mid-section mid-section-props])
+     [rn/view {:style (assoc
```

Comment:
why both merge and assoc ? you could use only assoc

Reply:
Because assoc expects kv pairs and merge will return a map .. I thought having common styles in variables would make things shorter .. Like Centrify

Reply:
Fixed